src/lib/sdl_renderer.cpp

```diff
@@ -74,6 +74,9 @@ namespace aronnax {
     r.w = int(box.x);
     r.h = int(box.y);
 
+    SDL_SetRenderDrawBlendMode(renderer_, SDL_BLENDMODE_BLEND);
```

Comment:
It'd be ideal if this was only called when required. But does this have to be reset after the call?